src/containers/other/NFTView/NFTView.tsx

```diff
@@ -121,27 +188,29 @@ const NFTView: React.FC<NFTViewProps> = ({ onTransfer }) => {
 	return (
 		<div className={styles.NFTView}>
 			<NFT
-				title={domainMetadata?.title as string}
 				owner={znsDomain?.owner.id as string}
+				title={domainMetadata?.title as string}
+				assetUrl={assetUrl as string}
 				creator={znsDomain?.minter.id as string}
+				options={nftMoreOptions}
+				description={znsDomain?.description as string}
 				onDownload={downloadAsset}
 				onShare={shareAsset}
-				assetUrl={assetUrl as string}
-				description={znsDomain?.description as string}
-				options={nftMoreOptions}
-				onSelectOption={onSelectNFTMoreOption}
+				onSelectOption={onSelectOption}
 			/>
 
 			<Actions
 				domainId={znsDomain?.id}
 				highestBid={highestBid}
 				buyNowPrice={buyNowPrice}
+				onMakeBid={openMakeABid}
+				onViewBids={openBidList}
```

Comment:
It would be nice to do this for each of the buttons but if the buttons are re-used in the codebase they would require a modal provider.